### DISCUSSION TITLE
opentelemetry-log crate: A minimal and simple OpenTelemetry log adapter

### DISCUSSION THREAD

**Discussion by vincenzopalazzo** [open-telemetry/opentelemetry-rust#2163] [Show and tell]
https://github.com/vincenzopalazzo/opentelemetry-log/releases/tag/v0.1.0

**Comment by cijothomas** [comment]
Is that any different than the https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-appender-log ?

**Reply by vincenzopalazzo** [reply]
Slightly, do not require setting up the appended and all the things.

Plus, in the future with a couple of compiler feature you can switch from `log` to `tracing` without changing 
the application log.

All code required is 

```rust
use opentelemetry_common::Opentelemetry;

fn main() {
    let mut manager = Opentelemetry::new();
    manager.init_log("example", &args.level, &url)?;
    // Your application code
}
```

So probably it is more hight level.
